src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-name/table-interactor-name.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+  <div class="nameAndIcons" [ngClass]="{inlineDisplay: interactorType==='stable complex'}">
+    <i *ngIf="organismIconDisplay"
+       class="{{interactorOrganismIcon}}"
+       title="{{interactorOrganism}}"></i>
+
+    <i *ngIf="interactorTypeDisplay"
+       class="{{interactorTypeIcon}}"
+       title="{{interactorType}}"></i>
+
+    <div class="name" title="{{interactorName}}" *ngIf="IDDisplay"> {{ interactorName }}
+    </div>
+    <div class="name" title="More information about {{interactorId}}" *ngIf="!IDDisplay">
+      <a *ngIf="!IDDisplay" href="{{identifierLink}}" target="_blank"
+         class="externalLinkContainer">
+        {{ interactorName }}
+      </a>
+    </div>
+
+    <a [routerLink]="['/complex/search']"
+       [queryParams]="{query: interactorId, page: 1}"
+       [fragment]="'view_complex_navigator'"
```

Comment:
Instead of `[fragment]="'view_complex_navigator'"`, we can use `[preserveFragment]="true"` to keep the same fragment we currently have in the URL. That way, if we rename it, then it does not matter.